packages/suite/src/components/wallet/AccountsMenu/components/AccountGroup/index.tsx

```diff
@@ -76,16 +81,15 @@ export default forwardRef((props: Props, _ref: React.Ref<HTMLDivElement>) => {
                         data-test={`@account-menu/${props.type}`}
                     >
                         <Translation id={getGroupLabel(props.type)} />
-                        {!props.keepOpened && (
-                            <Icon
-                                data-test="@account-menu/arrow"
-                                canAnimate={animatedIcon}
-                                isActive={isOpened}
-                                size={16}
-                                color={theme.TYPE_LIGHT_GREY}
-                                icon="ARROW_DOWN"
-                            />
-                        )}
+
```

Comment:
Why did you remove the condition? Isn't it confusing especially with the hover effect, if it does nothing (it's not possible to close the group)?

Reply:
Because it's better to see the `expanded` state, with the icon upside down, than to remove the icon completely. We don't remove it in Select :D And I changed the hover effect. 

22598792797b8c4022b75724baf2ef5f6bef4842